tracker-linear: module not found for @composio/core

## Bug

`pnpm dev` fails with a module resolution error in `tracker-linear`:

```
../plugins/tracker-linear/dist/index.js
Module not found: Can't resolve '@composio/core' in '/Users/dj/sj/agent-orchestrator/packages/plugins/tracker-linear/dist'
```

The package name should be `@composio/ao-core`, not `@composio/core`. Likely a stale or incorrect import in the tracker-linear plugin source.

--- a/packages/web/next.config.js
+++ b/packages/web/next.config.js
@@ -1,5 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  serverExternalPackages: ["@composio/core"],
   transpilePackages: [
     "@composio/ao-core",
     "@composio/ao-plugin-agent-claude-code",

